src/docs/background/payment-methods-overview/index.md

```diff
@@ -0,0 +1,121 @@
+
+![Card payment flow](card-payment-flow.png 'Card payment flow in Flex.')
+
+The preauthorization is valid for seven days after which Stripe
+automatically releases it. You can use the preauthorization to configure
+the transaction process to allow the provider to accept or decline the
+transaction. If the provider declines the transaction at this point,
+Stripe issues no fees regarding the transaction since no money has yet
+moved.
+
+Finally, the transaction process can hold money in Stripe until an
+explicit payout (The hold should not exceed three months). The mechanism
+allows you to build a transaction process where the money is in an
+escrow-like hold until an explicit payout. The hold ensures some safety
+for the customer as the marketplace controls the money until a payout.
+The process can issue the payout in a separate transition.
+
+### Push payment flow
+
+For synchronous push payments, there is no preauthorization. The flow
+starts from a synchronous charge. The charge moves money immediately to
+Stripe and there to the provider's connected account.
+
+![Push payment flow](push-payment-flow.png 'Push payment flow in Flex.')
+
+This means that refunds can be issued only at a point where money has
+already moved and the marketplace must cover Stripe fees. You should
+take this into account when designing the transaction process of your
+marketplace. Either declining the transaction should be disabled,
+handled through availability management, or accounted for in
+commissions.
+
+The rest of the money flow is the same as with card payments and has the
+same features and capabilities.
+
+## Using different payment methods in the transaction process
+
+The
+[default transaction process](https://github.com/sharetribe/flex-example-processes/tree/master/flex-default-process)
+in Flex supports card payments. The general article on
+[the transaction process](../transaction-process) describes the process
+in more detail.
+
+> **A note on Google Pay and Apple Pay**
+>
+> Even though Google Pay and Apple Pay are comparable to card payments,
+> they require some changes to the default implementation of FTW. To
+> enable them in FTW, you need to follow the
+> [Stripe instructions on the Request Payment Button](https://stripe.com/docs/stripe-js/elements/payment-request-button).
+
+If you wish to enable push payments, you need to change your transaction
+process and include new transitions. These should initiate the push
+payment and potentially skip the preauthorization phase. When you
+request a new push payment, you need to use the new action
+[`stripe-create-payment-intent-push`](../references/transaction-process-actions/#actionstripe-create-payment-intent-push).
+The example below describes the minimum recommended changes in the two
+transitions: `request-payment-push` and `confirm-payment-intent`.
+
+![Push payment process](push-payment-process.png 'Push payment process example.')
+
+## Further reading
```

Comment:
I think we could also add a short section explaining that when implementing the new payment methods, the Stripe's own instructions on how to deal with them with payment intents apply and the parts where something is done from server (basically just the payment intent creation), that's covered by Flex and remind that Flex exposes the payment intent ID and client secret through the transaction protected data to the customer.